lib/components/Utilities/Icon/index.tsx

```diff
@@ -7,7 +7,7 @@ import {
   SVGProps,
 } from "react"
 
-const iconVariants = cva("inline-block shrink-0", {
+const iconVariants = cva("inline-block shrink-0 text-f1-icon", {
```

Comment:
Sorry to jump in, since I don't have much context, but, wouldn't it be better to let the icon "inherit" the text color from the container where it is used by default?

That will make it easier for it to adapt to different states like hover/focus without having to pass down any more classes to this component

Reply:
It does not prevent you from overriding the default color and adapt the icon to your needs.

The issue with inheriting the text color is that in all figma files we have, the icon color is slightly lighter than the color of the main text. Since this is a subtle difference, it is easy to overlook and miss it. I noticed the color difference in this component, but there are several components ignoring the difference. This default class is a way to tackle the problem. 

What would be your suggestion? Also @josepjaume, maybe you have an opinion here too

Reply:
Yeah, discussed this with @dani-moreno and had the exact same concerns as you @rubenmoya. An option here would be to use `filter: darken(75%)` or comething like that? In tailwind, that'd be `brightness-75`.

Reply:
ok, will adjust the class

Reply:
Dammit